bin/make_run_ap_pipe_conf.py

```diff
@@ -0,0 +1,24 @@
+import os
+'''Generate configuration file for ap_pipe slurm run
+'''
+
+BASE_DIR = '/project/mrawls/hits2015/'
+shell_script = os.path.join(BASE_DIR, 'run_ap_pipe.sh')
+
+visits = [410915, 410929, 410931, 410971, 410985, 410987,
+          411021, 411035, 411037, 411055, 411069, 411071, 411255, 411269, 411271,
+          411305, 411319, 411321, 411355, 411369, 411371, 411406, 411420, 411422,
+          411456, 411470, 411472, 411657, 411671, 411673, 411707, 411721, 411724,
+          411758, 411772, 411774, 411808, 411822, 411824, 411858, 411872, 411874,
+          412060, 412074, 412076, 412250, 412264, 412266, 412307, 412321,
+          412324, 412504, 412518, 412520, 412554, 412568, 412570, 412604,
+          412618, 412620, 412654, 412668, 412670, 412704, 412718, 412720,
+          413635, 413649, 413651, 413680, 413694, 413696,
+          415314, 415328, 415330, 415364, 415378, 415380,
+          419791, 419802, 419804, 421590, 421604, 421606]
+
+filter = 'g'
```

Comment:
filter is a Python builtin function, so I'd suggest a different variable name here.